src/graphics/graphics.cpp

```diff
@@ -233,14 +233,21 @@ bool GraphicsManager::setFSAA(int level) {
 	return _fsaa == level;
 }
 
+/**
+ * Setup SDL + OpenGL renderer. We want OpenGL 3.2 for shader-based
+ * rendering and OpenGL 2.1 for the legacy renderer.
+ */
 bool GraphicsManager::setupSDLGL() {
-	if (WindowMan.initRender(_renderType, _debugGL, _fsaa))
-		return true;
-
-	// OpenGL 3.2 context not created. Spit out an error message, and try a 2.1 core context.
-
-	warning("Your graphics card hardware or driver does not support OpenGL 3.2. "
```

Comment:
Pedantic comment: strictly speaking the hardware/driver does not support OpenGL 3.2 (compat profile). Compat support is relatively recent in Mesa for example, but it's had GL3.2 core profile support for quite a while.